lib/aerospike/policy/write_policy.rb

```diff
@@ -74,6 +74,20 @@ def initialize(opt={})
       # Valid for Aerospike Server Enterprise Edition 3.10+ only.
       @durable_delete = opt.fetch(:durable_delete, false)
 
+      @records_per_second = opt[:records_per_second] || 0
```

Comment:
No documentation here?

Reply:
I moved records_per_second to statement class as it’s the same in the Java client